clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -1530,10 +1530,10 @@ static bool shouldAssumeDSOLocal(const CIRGenModule &cgm,
 
   const llvm::Triple &tt = cgm.getTriple();
   const CodeGenOptions &cgOpts = cgm.getCodeGenOpts();
-  if (tt.isWindowsGNUEnvironment()) {
-    // In MinGW, variables without DLLImport can still be automatically
-    // imported from a DLL by the linker; don't mark variables that
-    // potentially could come from another DLL as DSO local.
+  if (tt.isOSCygMing()) {
```

Comment:
Similar change should probably be applied in `clang/lib/CodeGen/CodeGenModule.cpp` as well?

Reply:
I think that has already been done, in #139797